test/unit/remote-config/remote-config.spec.ts

```diff
@@ -596,6 +614,18 @@ describe('RemoteConfig', () => {
           }).to.throw(
             'Cannot set property etag of #<RemoteConfigTemplateImpl> which has only a getter');
 
+          const v = template.version;
+          expect(v).to.be.not.undefined;
+          const version = v as Version;
```

Comment:
Why is this needed? Shouldn't this already be typed `Version`?

Reply:
If I do not cast it here the compiler complains that the object might be undefined. I think it is because of version is optional. We do the same for conditions below...
`const cond = c as RemoteConfigCondition;` any thoughts?

Reply:
Add an assertion.

```
const version = template.version!;
```